app/controllers/scheduled_messages_controller.rb

```diff
@@ -38,6 +38,6 @@ def update
   private
 
   def scheduled_message_params
-    params.require(:scheduled_message).permit(:body, :days_after_start, :title)
+    params.require(:scheduled_message).permit(:body, :days_after_start, :title, :tag_list)
```

Comment:
Yes!  I don't know why I missed that, I'm usually a stickler for such things myself.